third_party/blink/renderer/core/loader/document_loader.cc

```diff
@@ -2107,6 +2107,11 @@ void DocumentLoader::CommitNavigation() {
 
   WillCommitNavigation();
 
+  // if (!Url().ProtocolIsAbout() && !Url().ProtocolIsData() && !Url().ProtocolIs("chrome")) {
```

Comment:
What do you guys prefer?

Reply:
The way you have it now is good.  Explicitly only allowing HTTP is better than trying to disallow known other stuff.